### DISCUSSION TITLE
Is this feature to increase availability of containers exist in CRI-O?

### DISCUSSION THREAD

**Discussion by alpana17** [cri-o/cri-o#8020] [Q&A]
One of the important security triads is availability. Setting the --live-restore flag within the Docker daemon ensures that container execution is not interrupted when it is not available. This also makes it easier to update and patch the Docker daemon without application downtime.

Is there some similar feature in CRI-O?

**Comment by alpana17** [comment]
@saschagrunert Please suggest

**Comment by alpana17** [comment]
@saschagrunert Please suggest

**Comment by haircommander** [comment]
We have a similar mechanism out of the box actually. You can restart cri-o and not interrupt existing containers, and cri-o will restore them on startup. One limitation is we do cancel existing exec sessions.
